telegram/ext/filters.py

```diff
@@ -949,19 +952,103 @@ class _Private(MessageFilter):
         name = 'Filters.private'
 
         def filter(self, message: Message) -> bool:
+            warnings.warn(
+                'Filters.private is deprecated. Use Filters.chat_type.private instead.',
+                TelegramDeprecationWarning,
+                stacklevel=2,
+            )
             return message.chat.type == Chat.PRIVATE
 
     private = _Private()
-    """Messages sent in a private chat."""
+    """
+    Messages sent in a group or a supergroup chat.
```

Comment:
```suggestion
    Messages sent in a private chat.
```
uhm no its send in a private chat?